test/chunking-form/samples/missing-export/_config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+	description: 'missing export',
+	options: {
+		input: ['main.js', 'dep2.js'],
+		plugins: [
+			{
+				missingExport() {
+					return true;
+				}
```

Comment:
Rather than have the missingExport hook provide this functionality, how about making this a flag option - `shimMissingExports: true`?

Reply:
That is already the way to stop the default behavior of hard erroring (in non-code-splitting mode). Are you suggesting checking [here](https://github.com/rollup/rollup/blob/v0.57.1/src/utils/defaults.ts#L76) for `shimMissingExports` and conditionally not erroring?

Reply:
Yes exactly, so that this can be done without a plugin.

Reply:
done